src/App.js

```diff
@@ -1,40 +1,54 @@
-import { BrowserRouter, Route, Routes} from "react-router-dom"
+import { BrowserRouter, Link, Route, Routes} from "react-router-dom"
 import Container from 'react-bootstrap/Container';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
 import Navbar from 'react-bootstrap/Navbar';
 import Nav from 'react-bootstrap/Nav';
+import LinkContainer from 'react-router-bootstrap/LinkContainer';
+import { useState } from 'react';
 
+const langs = [{language:"C", resources:["Harvard's CS50X"]}, {language:"Python 3", resources:["Python Crash Course", "Harvard's CS50X", "Harvard's CS50P", "Code Combat", "The Big Book of Small Python Projects"]}, {language:"HTML/CSS", resources:["Harvard's CS50X", "Harvard's CS50W", "FreeCodeCamp"]}, {language:"JavaScript", resources:["Harvard's CS50W", "Mozilla Developer Network", "Code Combat"]}, {language:"SQL", resources:["Harvard's CS50X"]}, {language:"Django", resources:["Harvard's CS50W"]}, {language:"React", resources:["Harvard's CS50W", "React Documentation"]}];
```

Comment:
this is probably a good place to plug a code linter, like Prettier or ESLINT with a default set of plugins. A one liner with a bunch of data like this is tough to grok.

In VSCode you can open the command palette to manually lint, or set it up to auto lint and format when saving files.